src/parsers/index.js

```diff
@@ -74,8 +75,17 @@ const processStyledComponentsFile = (ast, absolutePath, options) => {
       extractedCSS.push(stylelintCommentsAdded)
       sourceMap = Object.assign(
         sourceMap,
-        getSourceMap(extractedCSS.join('\n'), wrappedContent, processedNode.loc.start.line)
+        getSourceMap(extractedCSS.join('\n'), wrappedContent, processedNode.quasi.loc.start.line)
       )
+      // Save interpolation lines
+      node.quasi.expressions.forEach((expression, i) => {
+        let l = node.quasi.quasis[i].loc.end.line + 1
+        while (l <= node.quasi.quasis[i + 1].loc.start.line) {
```

Comment:
Doesn't this throw an error when `i === node.quasi.expressions.length - 1` ??

Reply:
No, `quasis.length` should be always greater 1 than `expressions.length`. I learned from similar codes [here](https://github.com/styled-components/stylelint-processor-styled-components/blob/master/src/utils/tagged-template-literal.js#L122).

Reply:
Ahh yes, you're iterating over expressions not quasis. Makes sense, my bad!

Reply:
It'd be an okay idea to add a comment about that, not super urgent, as it's not that unclear, but no harm in documenting code well!